content/blog/2023/05/camel-debezium-quarkus/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Change Data Capture with Apache Camel and Debezium"
+date: 
+draft: true
```

Comment:
Just a heads up ... You'll probably need to change this to false, after the review.

Reply:
Changed in recent commit